network-api/networkapi/buyersguide/templates/contest.html

```diff
@@ -0,0 +1,373 @@
+          </p>
+          <p>
+            Entries must be manually key-stroked by the entrant; use of automated entry devices or programs, or entries by third parties
+            by any method of entry, are prohibited. Entries that do not conform to or that violate these Official Rules or are submitted
+            in any manner other than stated in these Official Rules will be disqualified. Sponsor will not be responsible for late, lost,
+            inaccessible/blocked, incomplete or misdirected entries. Proof of submission (posting) of Entry does not constitute proof of
+            receipt of same. All Entries must be submitted in their entirety prior to expiration of the Contest Period. By entering this
+            Contest, entrant agrees to these Official Rules and the decisions of the Sponsor, which are binding and final in all matters
+             relating to this Contest.
+          </p>
+          <p>
+            You may only use one (1) Twitter or Instagram account to participate in the Contest. All Tweets and replies must comply with
+            the Twitter or Instagram Terms of Service and Twitter or Instagram Rules available at the Twitter or Instagram Websites.
+            <strong>Important note:</strong> If your Twitter account is set to “protected mode,” or “private,” your Entry may not be
+            viewable by Sponsor, and thus, it is recommended that your Twitter account is set to “public” in order to enter.
+          </p>
+          <p>
+            If you opt to access Twitter or Instagram via your wireless mobile device, data rates may apply for each message sent or
+            received from your handset according to the terms and conditions of your service agreement with your wireless carrier.
+            Other charges may apply (such as normal airtime and carrier charges as well as charges for wireless Internet access) and
+            may appear on your mobile phone bill or be deducted from your prepaid account balance. Wireless carriers’ rate plans may vary,
+            and you should contact your wireless carrier for more information on messaging rate plans and charges relating to your
+            participation in this Contest. Mobile device service may not be available in all areas. Check your phone’s capabilities
+            for specific instructions.
+          </p>
+        </section>
+
+
+        <section aria-labelledby="guidelines">
+          <p id="guidelines"><strong>3. Content Guidelines.</strong></p>
+          <p>
+            In addition to complying with all other requirements of these Official Rules and/or on the and/or on Twitter or Instagram;
+            each Entry must comply with the following guidelines (<strong>“Content Guidelines”</strong>). Any Entry that Sponsor, in its
+            sole discretion, determines is in violation of any Content Guideline may be disqualified.
+          </p>
+          <p>
+            For all entries:
+          </p>
+          <ul class="body">
```

Comment:
![image](https://user-images.githubusercontent.com/2896608/100283011-83682980-2f21-11eb-9a35-6bedeaa801f3.png)

According to the mockup, this list should not have bullets. You can add `list-unstyled` to `<ul>` to unstyle it.

```suggestion
          <ul class="body list-unstyled">
```

Reply:
it should: the original doc had them, and they were forgotten in the comp. I asked Kristina whether to omit or include them and got signoff on making sure they were there =)